src/InjectOpenGLIntrinsics.cpp

```diff
@@ -40,33 +40,41 @@ class InjectOpenGLIntrinsics : public IRMutator {
             //                   c - c_min, c_extent
             //                   )
             //
+            int dims = (call_args.size() - 2) / 2;
+            internal_assert(dims >= 1 && dims <= 3);
+
             vector<Expr> args(5);
             args[0] = call_args[0];  // "name"
             args[1] = call_args[1];  // name.buffer
 
             // Normalize first two coordinates.
-            for (size_t i = 0; i < 2; i++) {
+            for (size_t i = 0; i < std::min(dims, 2); i++) {
```

Comment:
This triggers

```
InjectOpenGLIntrinsics.cpp:51:34: error: comparison between signed and unsigned integer expressions [-Werror=sign-compare]

             for (size_t i = 0; i < std::min(dims, 2); i++) {
```

For some compilers; please fix.

Reply:
It's probably simplest to rewrite as `for (int i ...)`.

Reply:
Thanks! Sorry for that...